net/xia/list_fib.c

```diff
@@ -171,6 +171,12 @@ static void list_fxid_init(struct fib_xid *fxid, int table_id, int entry_type)
 	fxid->dead.xtbl = NULL;
 }
 
+static void list_fxid_copy(struct fib_xid *old_fxid, struct fib_xid *new_fxid)
+{
+	memcpy(new_fxid, old_fxid,
```

Comment:
Although it's unlikely that `new_fxid == old_fxid`, it would be safer to replace `memcpy()` with `memmove()`.